packages/frontend-2/assets/css/tailwind.css

```diff
@@ -51,15 +52,18 @@
   .caption {
     @apply text-xs;
   }
-}
 
-/**
- * Ensuring ring offset color always defaults to bg color
- */
-*,
-[type='checkbox']:focus,
```

Comment:
Why did you remove the ring offset background styles? 🤔 If these aren't set, then you get ugly white ring offsets in the dark theme. Are we scrapping ring offsets altogether as a design choice? If so, then I guess it makes sense to remove this